development.ini

```diff
@@ -4,7 +4,7 @@
 ###
 
 [app:main]
-use = egg:cnx-port
+use = egg:cnx-cnxpublishingbuilds
```

Comment:
Typo. :stuck_out_tongue_winking_eye:  This would be ``egg:cnx-publishing-builds``. The distribution is named cnx-publishing-builds and the package is cnxpublishingbuilds.